uvloop/includes/stdlib.pxi

```diff
@@ -63,6 +63,8 @@ cdef gc_disable = gc.disable
 cdef iter_chain = itertools.chain
 cdef inspect_isgenerator = inspect.isgenerator
 
+cdef int IPV6_V6ONLY = getattr(socket, 'IPV6_V6ONLY')
```

Comment:
Is this the right place / way to do it?

Reply:
It is, but `IPV6_V6ONLY` might be missing on some platforms.  So you probably want to add two constants: `has_IPV6_ONLY = hasattr(socket, 'IPV6_V6ONLY')` and `IPV6_V6ONLY = getattr(socket, 'IPV6_V6ONLY', -1)`

Reply:
Context: this whole ugly file is the result of Cython not cleaning up its PyModules properly.  That lead to circular references that prevented uvloop ResourceWarnings from showing (say about an event loop that the user forgot to properly close).  So this file is a workaround for that bug -- it references stuff from the stdlib without keeping references to Python modules.

Now this Cython shortcoming has been fixed in some recent Cython release, so we probably can start importing things properly. It's something for us to check later.

Reply:
OK

Reply:
No, it's the same. It's just an inconsistency that we'll want to fix at some point.  Initially my idea was to use Python constants for Python objects/APIs and `uv.*` constants for libuv, but I guess it's actually more confusing this way.